.gitignore

```diff
@@ -3,3 +3,17 @@ src/*/target
 src/*/Cargo.lock
 src/libs/protocols/src/image.rs
 src/libs/protocols/src/image_ttrpc.rs
+
+tools/debug/Cargo.lock
+tools/debug/target/
+tools/debug/protocols/asynchronous/empty.rs
+tools/debug/protocols/asynchronous/image.rs
+tools/debug/protocols/asynchronous/image_ttrpc.rs
```

Comment:
This pr does not contain these files

Reply:
The changes are from PR #65, this PR will be rebased on it in the future.